internal/provider/datasources/data_source_team_test.go

```diff
@@ -0,0 +1,49 @@
+package datasources_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/astronomer/terraform-provider-astro/internal/utils"
+
+	astronomerprovider "github.com/astronomer/terraform-provider-astro/internal/provider"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAcc_DataSourceTeam(t *testing.T) {
+	teamId := os.Getenv("HOSTED_TEAM_ID")
```

Comment:
once we add team resource we should create a team using it instead of an existing team